tests/snapshot_fail_test.c

```diff
@@ -40,10 +40,11 @@ char* SOCKET_PORT = "SOCKET_PORT";
  * A simple test to verify that snapshots fail if the following are found in the
  * payload being snapshotted.
  * - epoll fd's with events
- * - pipes with queued data
- * - socketpair's with queued data
  * - sockets that are in a connected state.
  * - interval timers that are running (setitimer() and timer_create())
+ * Now the following conditions are supported.  So test that they work.
+ * - pipes with queued data
+ * - socketpair's with queued data
```

Comment:
Kind of contradicts the name of the test. No bug deal, just a bit confusing

Reply:
Yes, I mentioned this to you in our conversation about recovering "pipe" contents before this change was created.
The test should probably be renamed to snapshot_cornercase_test.